tests/remote_attestation/maa/oe_enclave/enc/private.pem

```diff
@@ -0,0 +1,39 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Yes this is needed for signing oe_enclave app.
oe_enclave/enc/makefile:   oesign sign -e oeApp_enc -c enc.conf -k private.pem

Reply:
We really shouldn't check in any private/public keys if they are not necessary. The SGX-LKL build already sets up a random key pair that can be reused, or the key generation can be duplicated to generate different ones.

Reply:
Checked this with @jxyang  as well. private.pem is private key for signing oe_enclave app to be verified by sgxlkl_enclave against a MR signer. If we generate random key pairs, the test and verification will fail. That is why we need a preknown MRSIGNER and need to sign oe_enclave with matching private key (private.pem) so that verification can pass and TLS connection can be established. So this is required.